Src/AutoFakeItEasy.FakeItEasy4UnitTest/AutoFakeItEasy.FakeItEasy4UnitTest.csproj

```diff
@@ -17,7 +17,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FakeItEasy" Version="[4.0.0]" />
+    <PackageReference Include="FakeItEasy" Version="[4.0.0]" Condition=" '$(TargetFramework)'=='net452' " />
+    <PackageReference Include="FakeItEasy" Version="[4.0.0]" Condition=" '$(TargetFramework)'=='netcoreapp1.1' " />
+    <PackageReference Include="FakeItEasy" Version="[4.9.0]" Condition=" '$(TargetFramework)'=='netcoreapp2.0' " />
```

Comment:
Hey, @zvirja!
You'd asked for 4.9.2 (which I can still do; just say the word) but 4.9.0 is the minimum, and I thought it would be more consistent to use that version.
Suits?

Reply:
@blairconrad Sure, I just messed up the things, sorry 😖 Please go with `4.9.0`.

Reply:
I don't think _you_ messed the things up. I messed the things up (by deviating from your practices) and you are putting me back on track!

Reply:
Well, I'm not taking the whole responsibility😅 My contribution is limited by `4.9.2` vs `4.9.0` only, while all the rest is still supposed to be caused solely by you 🤣 

Merged the code, so you are enabled in #1088.

Good night 🌔

Reply:
Thanks. g'night!